### PR TITLE
added navbar, update the landing page with additional content

### PR DESCRIPTION
**Pull request by gc824**
Team worked on cleaning up the Pull Request. We also added sticky header in CSS.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,41 +1,35 @@
-               body {
-                   background-image: url(../img/PizzaParty.jpg);
-                   height: 100%;
-                   background-position: center;
-                   background-repeat: no-repeat;
-                   background-size: cover;
-                   position: relative;
-               }
+                body {
+                	background-image: url(../img/PizzaParty.jpg);
```
Comment (billythakidd04) [inline css/style.css:2]: the image path should not include the `../`

File: css/style.css [hunk 2]
```diff
@@ -1,41 +1,35 @@
...
 
-               h1 {
-                   color: rgb(201, 24, 24);
-                   font-family: "Arial";
-                   font-size: 24px;
-               }
+                h1 {
+                	color: rgb(201, 24, 24);
+                	font-family: "Arial";
+                	font-size: 2410px;
```
Comment (billythakidd04) [inline css/style.css]: thats pretty immense...

File: index.html [hunk 3]
```diff
@@ -1,19 +1,127 @@
 <!DOCTYPE html>
 <html>
-
 <head>
 	<title>PizzaParty</title>
 	<link href="css/style.css" rel="stylesheet">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
```
Comment (billythakidd04) [inline index.html]: these two lines need to switch per https://getbootstrap.com/docs/5.0/getting-started/introduction/#css

File: index.html [hunk 4]
```diff
@@ -1,19 +1,127 @@
...
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (billythakidd04) [inline index.html]: this needs to be before the css

File: index.html [hunk 5]
```diff
@@ -1,19 +1,127 @@
...
 </head>
-
 <body>
-	<h1>Pizza Party</h1>
-		<div class="topnav">
-			<a href="ourstory.html ">Our Story</a>
-			<a href="Menu.html ">Menu</a>
-			<a href="contactus.html ">Connect with Us!</a>
-			<a href="Recipe.html">Recipe</a>
+				<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```
Comment (billythakidd04) [inline index.html]: do you need _all_ of those classes?

File: index.html [hunk 6]
```diff
@@ -1,19 +1,127 @@
...
+				<div class="container-fluid">
+					<a class="navbar-brand" href="#">PizzaParty</a>
+				  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDarkDropdown" aria-controls="navbarNavDarkDropdown" aria-expanded="false" aria-label="Toggle navigation">
+					<span class="navbar-toggler-icon"></span>
+				  </button>
```
Comment (billythakidd04) [inline index.html]: Is this functional?

File: index.html [hunk 7]
```diff
@@ -1,19 +1,127 @@
...
+						  <li><a class="dropdown-item" href="#">Specialty Pizzas</a></li>
+						</ul>
+					  </li>
+					</ul>
+				  </div>
+				</div>
+			  </nav>
 		</div>
-</body>
+	</nav>
```
Comment (billythakidd04) [inline index.html]: extraneous closing tags

File: menu.html [hunk 8]
```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
-
 <head>
 	<title>PizzaParty</title>
 	<link href="css/style.css" rel="stylesheet">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
```
Comment (billythakidd04) [inline menu.html]: swap these two lines per https://getbootstrap.com/docs/5.0/getting-started/introduction/#css

File: menu.html [hunk 9]
```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
-
 <head>
 	<title>PizzaParty</title>
```
Comment (billythakidd04) [inline menu.html]: this should probably say PizzaParty - Menu

File: menu.html [hunk 10]
```diff
@@ -1,9 +1,9 @@
...
 	<link href="css/style.css" rel="stylesheet">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
 </head>
 <meta charset="UTF-8">
```
Comment (billythakidd04) [inline menu.html]: this should be in the head or removed completely

File: menu.html [hunk 11]
```diff
@@ -1,9 +1,9 @@
...
 <title>Menu</title>
```
Comment (billythakidd04) [inline menu.html]: Is this supposed to be here? If so where is the opening body tag

File: ourstory.html [hunk 12]
```diff
@@ -1,27 +1,30 @@
 <!DOCTYPE html>
 <html>
-
 <head>
     <title>PizzaParty</title>
```
Comment (billythakidd04) [inline ourstory.html]: PizzaParty - Our Story

File: ourstory.html [hunk 13]
```diff
@@ -1,27 +1,30 @@
...
+    <link href="css/style.css" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
```
Comment (billythakidd04) [inline ourstory.html]: swap per https://getbootstrap.com/docs/5.0/getting-started/introduction/#css

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
This pr needs A LOT of work. 90% of these changes should have never been in this pull request. Don't add semi finished work to a pr. Also there needs to be a lot of formatting fixes made and there are still a ton of tags that aren't close or duplicated. Please fix these issues before requesting another review.

**Comment by billythakidd04** [comment]
@gc824, there is NO description, NO assignee, and more than half of this pr is unrelated or incomplete. _AND_ there are conflicts... We can't have pull requests like this.

File: css/style.css (hunk 1 above)
Comment (billythakidd04) [reply css/style.css:2]: I was wrong disreagard

File: index.html (hunk 6 above)
Comment (cwlainson) [reply index.html]: Yes it is functional, it is a toggler that converts to a hamburger style when the screen size changes.

File: index.html (hunk 7 above)
Comment (cwlainson) [reply index.html]: all extra tags have been removed

File: menu.html (hunk 10 above)
Comment (cwlainson) [reply menu.html]: meta has been moved inside head tag

File: menu.html (hunk 11 above)
Comment (cwlainson) [reply menu.html]: line 9 removed

File: ourstory.html (hunk 12 above)
Comment (cwlainson) [reply ourstory.html]: made changes

File: ourstory.html (hunk 13 above)
Comment (cwlainson) [reply ourstory.html]: swapped lines correctly

File: menu.html (hunk 9 above)
Comment (cwlainson) [reply menu.html]: fixed title with PizzaParty - Menu

File: menu.html (hunk 8 above)
Comment (cwlainson) [reply menu.html]: lines switched

File: ourstory.html (hunk 12 above)
Comment (cwlainson) [reply ourstory.html]: fixed title with PizzaParty - Our Story

File: contactus.php [hunk 14]
```diff
@@ -1,20 +1,58 @@
...
+            </div>
+        </div>
+    </nav>
     <main>
         <h1> Connect with Us! </h1></br>
         <p>Send E-mail</p>
-        <form class = "contact-form" action= "contactus.php" method="post">
+        <form class="contact-form" action="contactus.php" method="post">
             <input type="text" name="name" placeholder="Full Name">
             <input type="text" name="mail" placeholder="Your Email">
```
Comment (billythakidd04) [inline contactus.php:50]: this type should be email

File: index.html [hunk 15]
```diff
@@ -3,17 +3,127 @@
...
+		<p>I love when it's Hot Bread in the oven</p>
+		<hr>
+
+		<h4><b>Chris</b>$20.00</h4>
+		<p>Pepperoni, Bacon & Peppers.</p>
+		<hr>
+
+		<h1>Recipe</h1>
+		
+		</div>
```
Comment (billythakidd04) [inline index.html]: each of these menu items should be in their own container. possibly a bootstrap card or something of that nature

**Review by billythakidd04 — APPROVED** [review]
Few suggestions.

**Pull request merged**